src/main/java/org/launchcode/liftoffrecipetracker/Data/UserRepository.java

```diff
@@ -0,0 +1,9 @@
+package org.launchcode.liftoffrecipetracker.Data;
```

Comment:
IntelliJ says Data should not be capitalized.  Package name 'org.launchcode.liftoffrecipetracker.Data' does not correspond to the file path 'org.launchcode.liftoffrecipetracker.data'